librarian/views/filemanager/_main.tpl

```diff
@@ -48,15 +48,20 @@ def get_views(facet_types):
         ## - Link to complete file list if there is search query
         ## - Link to complete file list if invalid path is requested
         ## - Link to parent directory if no search query and not at top-level
+        <% h.urlquote %>
```

Comment:
Is this one of those peek-a-boo issues where it doesn't work unless you do this?